board/fischertechnik/TXT/rootfs/bin/calibrate-touchscreen

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+calibrate() {
+	sudo TSLIB_TSDEVICE=/dev/input/event0 /usr/bin/ts_calibrate
```

Comment:
Now that you have a real 'backend', I would drop the `sudo` here, and simply call the whole script with `sudo` from the GUI tool. and - since you went all the way to make it useable to users other than 'ftc', move the flag file back to `/etc` (which is not prblem any longer if you run the whole script as root anyway).

Maybe add a warning if you are not running as root.